scripts/game.js

```diff
@@ -89,7 +81,7 @@ class Game {
         x: 18,
         y: 1,
         size: "20px",
-        content: `${this.calculateTime()[1]} : ${this.calculateTime()[0]}`,
+        content: `${this.calculateTime().elapsedMinutes} : ${this.calculateTime().elapsedSeconds}`,
```

Comment:
Nice improvement!

Even better would be to create a class for it. :)

Something like:

```javascript
content: new GameTime().toString()
```

Reply:
I guess :p